src/main/java/com/faforever/client/config/BaseConfig.java

```diff
@@ -49,10 +51,19 @@ Locale locale() {
   MessageSource messageSource() {
     ResourceBundleMessageSource messageSource = new ResourceBundleMessageSource();
     messageSource.setBasename("i18n.messages");
+    messageSource.setFallbackToSystemLocale(false);
     return messageSource;
   }
 
   @Bean
+  ArrayList<String> languageInfo(){
+    String[] x={"en","UK","de","DE","ru","RU"};
+    ArrayList<String> languageInfo=new ArrayList<String>();
+    for (String s: x)
+    {languageInfo.add(languageInfo.size(),s);}
+    return languageInfo;
+  }
```

Comment:
Best would be to ready this kind of things out from the ResourceBoundle.... But I got no clue how to